### PR TITLE
Added Service Chain Network on homi docker compose

### PR DESCRIPTION
**Pull request by jeongkyun-oh**
- service chain network is added on docker compose

## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: cmd/homi/docker/compose/homi.go [hunk 1]
```diff
@@ -38,27 +38,29 @@ type Homi struct {
 	TxGenOpt          service.TxGenOption
 }
 
-func New(ipPrefix string, number int, secret string, addresses []string, nodeKeys []string,
-	genesis string, staticCNNodes string, staticPNnodes string, dockerImageId string, useFastHttp bool, networkId int,
-	useGrafana bool, proxyNodeKeys []string, enNodeKeys []string, useTxGen bool, txGenOpt service.TxGenOption) *Homi {
+func New(ipPrefix string, number int, secret string, addresses, nodeKeys []string,
+	genesis, serviceGenesis, staticCNNodes, staticPNNodes, staticENNodes, staticSCNNodes, bridegeNodes, dockerImageId string, useFastHttp bool, networkId, parentChainId int,
```
Comment (KimKyungup) [inline cmd/homi/docker/compose/homi.go]: How about serviceGenesis -> scGenesis? :)

**Comment by jeonghonah** [comment]
Please check gofmt.

File: cmd/homi/docker/compose/homi.go (hunk 1 above)
Comment (KimKyungup) [reply cmd/homi/docker/compose/homi.go]: or servicechainGenesis?

**Comment by KimKyungup** [comment]
@jeongkyun-oh Thanks for this PR. LGTM except gofmt and my comment. :)

**Comment by jeongkyun-oh** [comment]
@KimKyungup @jeonghonah PTAL.

**Pull request merged**